ci/configs/cosm-orc/testnet.yaml

```
@@ -2,7 +2,6 @@ chain_cfg:
   denom: "ujunox"
   prefix: "juno"
   chain_id: "uni-3"
```

```suggestion
  chain_id: "uni-5"
```

We are on `uni-5` now.

---

Good call. Will fix the code_ids too.